tests/Antlers/ScratchTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Tests\Antlers;
+
+use Facades\Tests\Factories\EntryFactory;
+use Statamic\Facades\Antlers;
+use Statamic\Facades\Collection;
+use Statamic\Structures\CollectionStructure;
+use Tests\PreventSavingStacheItemsToDisk;
+use Tests\TestCase;
+
+class ScratchTest extends TestCase
```

Comment:
I've added this class as a temporary place to dump tests for issues I've noticed, without needing to think too hard about moving them to the right place. We can move the tests out to better places later.